user-service/src/main/java/org/java/mentorship/user/exception/handler/ServiceExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.java.mentorship.user.exception.handler;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.java.mentorship.contracts.user.dto.UserServiceError;
+import org.java.mentorship.user.exception.domain.common.UserServiceException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ServiceExceptionHandler {
+
+    @ExceptionHandler
+    public ResponseEntity<UserServiceError> handleException(final UserServiceException ex, final HttpServletRequest request) {
```

Comment:
Dont' think you need the request parameter

Reply:
Here indeed, I didn't use it